gradlew

```diff
@@ -1,74 +1,129 @@
-#!/usr/bin/env bash
+#!/bin/sh
+
+#
+# Copyright © 2015-2021 the original authors.
```

Comment:
Where did these versions of gradlew and gradle-wrapper.jar come from? They're not the same as the ones generated by the Android Studio new project wizard, which apparently haven't changed since Arctic Fox.

Reply:
I doubled checked, indeed - they are different. What I just did is run Android Studio migration assistant to update versions. My thought is during migration from version to version it might brought this changes.
 
My proposal would be the same: if tests passes, it is sage to merge it into the master.

Reply:
I checked code on the emulator (API 33) and it compiles and works without noticed issues.

Reply:
Strange, I just tried the migration assistant myself, and it didn't change gradlew, gradlew.bat or gradle-wrapper.jar. So as above, I've reverted all three files to be more representative of what most users will have.